app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict: dict = {}
+
+    def wrapper(*args) -> int:
```

Comment:
The return type hint for the `wrapper` function is currently set to `int`. However, since this is a decorator, the return type should match the return type of the function it decorates. Consider changing the return type hint to `Any` or removing it altogether to make it more flexible.